sdk/storage/azblob/log.go

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azblob
+
+import "github.com/Azure/azure-sdk-for-go/sdk/storage/azblob/internal/exported"
+
+const (
+	// EventUpload is used when we compute number of blocks to upload and size of each block.
```

Comment:
I know that at present this only logs block computations.  However, we should generalize this doc comment in case this expands in the future.